docs/_articles/en/user-guide/perf-enhancements.md

```diff
@@ -20,4 +20,5 @@ In this beta release, performance enhancements are effective when you run the **
 - Apex Trigger
 - Visualforce Component
 - Visualforce Page
-- Lightning Components
+- Lightning Web Components
```

Comment:
@lcampos, now that performance enhancements cover all components (Apex, Visualforce, LWC and Aura), should we delete these lines from intro para:

Currently, these changes only support Apex and Visualforce metadata types. We’ll add support for more metadata types in future releases.

Reply:
Good catch, let me update it to something more generic and still keep that the coverage is limited but we'll be expanding it as time goes by.